### ISSUE TITLE
Please include the reduction-effects plugin in the standard install

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#9692]
This plugin is useful for debugging Gallina, but is extremely hard to discover (it took me three or four searches to find the PR that added support for it).  I just used it to [answer a discourse thread](https://coq.discourse.group/t/debugging-fixpoint-execution/183/2?u=jasongross).
The first step might be getting the plugin into opam and the CI.

cc @herbelin as the author of the plugin

**Comment by Zimmi48** [comment]
The reasons that pushed for this to be done as an external plugin were kind of dubious from the beginning, as this was not the choice of the author of the plugin, makes discoverability harder, etc. The experimental nature of the thing could instead be flagged as proposed in #9664.

Furthermore, the author is not used to having to maintain external plugins, and similarly to Bignums, we end up with everyone forgetting to update the opam package, etc.

I see two possible routes for these two plugins: either they get back into the main repo (this could be under an `external/` folder, which would be tested in CI in the same way that external plugins are tested, with specific `opam` files, etc.) or they move to coq-community. The Coq development team has no business maintaining a large diversity of packages spread out over multiple repositories.

**Comment by gares** [comment]
I think the simplest thing today is to have it part of coq-comunity, since the `external/` thing is not a thing yet, while coq-community is.

**Comment by Zimmi48** [comment]
Right. @JasonGross Would you like to be the plugin's principal maintainer in coq-community? Given that we can also add it to Coq's CI at the same time (as you suggested), this will mostly consist in handling incoming PRs, and preparing new compatibility releases every six months.

**Comment by Zimmi48** [comment]
(and we can keep this issue open until it is added to the Windows bundle)

**Comment by JasonGross** [comment]
Sure, I am fine with handling incoming PRs and branching / tagging / updating opam or whatever as necessary, every six months, as long as someone tells me what needs to be done the first time.  (I will probably write a script that executes the "prepare compatibility release" steps.)

**Comment by Zimmi48** [comment]
>I will probably write a script that executes the "prepare compatibility release" steps.

See also coq-community/manifesto#49.

**Comment by Zimmi48** [comment]
This plugin is now hosted in coq-community: https://github.com/coq-community/reduction-effects
Before it can be added to Coq's CI, it will need to be ported to Coq 8.9-8.11.

**Comment by liyishuai** [comment]
- [master](https://github.com/coq-community/reduction-effects/tree/master) branch is now tested with Coq 8.10, 8.11 and dev.
- [v0.1.2](https://github.com/coq-community/reduction-effects/releases/tag/v0.1.2) supports Coq 8.10 and 8.11 (and current dev).
- [v0.1.1](https://github.com/coq-community/reduction-effects/releases/tag/v0.1.1) supports Coq 8.9.
- [v0.1.0](https://github.com/coq-community/reduction-effects/releases/tag/v0.1.0) supports Coq 8.8.

**Comment by JasonGross** [comment]
Note that it's been in the Coq's CI since https://github.com/coq/coq/pull/11469.  Now we just need to get it included in the installer.

**Comment by JasonGross** [comment]
cc @MSoegtropIMC , I think

**Comment by MSoegtropIMC** [comment]
OK, I will add it.

**Comment by Zimmi48** [comment]
@MSoegtropIMC I can also address this if you don't have time.

**Comment by MSoegtropIMC** [comment]
It would help, yes.

**Comment by MSoegtropIMC** [comment]
Btw.: my plan is to deliver a Coq platform based Windows installer for 8.12 in the intended timeframe (1..3 months after the 8.12 release - it will likely be less than 2 months). I also plan to deliver a beta for 8.12 beta. So I don't know if it is required to add this to the 8.12 legacy Windows installer. I am currently concentrating on getting the dependency wise complicated pieces of the platform pulled straight (VST and CompCert).

**Comment by Zimmi48** [comment]
Let's delay this issue to the platform. We can always decide to add it in a patch-level release of Coq if the platform work hasn't progressed as much as expected.
